packages/block-editor/src/components/responsive-block-control/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+import { Fragment } from '@wordpress/element';
+
+import {
+	ToggleControl,
+} from '@wordpress/components';
+
+export function ResponsiveBlockControlLabel( { property, device } ) {
+	return (
+		<Fragment>
+			<span className="screen-reader-text">
+				{ property } for
```

Comment:
I don't think that's going to work for translations. We probably have to build the whole string for screen readers with `sprintf`, and expose it to screen readers in a different way (maybe `aria-label`?).

Reply:
Thanks! I thought I'd caught all these on my pass i18n pass, but due to this change to the component it's slipped through again.

I'll get this fixed as well.

Reply:
Fixed